docs/how-to-add-in-a-new-app.md

```diff
@@ -0,0 +1,4 @@
+# How to add in a new app
+
+
+If a new github repo is added, what all activities need to be done to deploy its code in production
```

Comment:
this is good - like a checklist of what's needed like folder for app, TF cloud workspace, github action to build container, github action to deploy container, permissions, etc.